docs/rules/arrow-body-style.md

```diff
@@ -89,7 +91,9 @@ let foo = () => ({ bar: 0 });
 
 #### requireReturnForObjectLiteral
 
-When the rule is set to `"as-needed", { requireReturnForObjectLiteral: true }` the following patterns are considered problems:
+> This option is only applicalble when used in conjunction with the `"as-needed"` option.
```

Comment:
Good spot on the typo - got that fixed in PR batch 4 that's queued behind this PR.